dashboard.go

```diff
@@ -122,20 +206,29 @@ func (c *Client) Dashboard(slug string) (*Dashboard, error) {
 	return result, err
 }
 
-func (c *Client) DeleteDashboard(slug string) error {
-	path := fmt.Sprintf("/api/dashboards/db/%s", slug)
+// DeleteDashboard deletes a grafana dashoboard
+func (c *Client) DeleteDashboard(uid string) (string, error) {
```

Comment:
Can we add a `DeleteDashboardByUid` function and mark the existing one as deprecated?